src/components/HEADER/Header.module.css

```diff
@@ -0,0 +1,199 @@
+body{
+	padding: 0px;
+	margin-left: 0px;
+}
+
+#head{
+	position: sticky;
+	top:0;
+	right: 0;
+}
+#nav{
+	padding-right: 10px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: rgb(91, 3, 3);
+	
+}
+
+#nav #title{
+	display: flex;
+	font-size: 1.5rem;
+	margin:1rem;
+	font-weight: bold;
+}
+
+#nav ul{
+	display: flex;
+}
+
+#nav ul li a{
+   display:block;
+	padding: 8px;
+	margin: 0 8px;
+	border-radius: 50px;
+}
+
+#nav #menu{
+	display: none;
+	position: absolute;
+	top:12px;
+	right: 8px;
+	flex-direction: column;
+	justify-content: space-between;
+	width:36px;
+	height: 32px;
+}
+
+#nav #menu span{
+	height: 6px;
+	width:100%;
+	background-color: rgb(255, 239, 174);
+	border-radius: 2px;
+}
+
+#login{
+	box-shadow: 5px 25px 50px 12px rgb(0 0 0 / 0.25);
+}
+#cart_mobile{
+	display: none;
+}
+
+
```

Comment:
tip for future tailwind + css/scss 
and apply 
https://tailwindcss.com/docs/using-with-preprocessors